app/controllers/sign_up/completions_controller.rb

```diff
@@ -84,7 +84,6 @@ def analytics_attributes(page_occurence)
         ialmax: resolved_authn_context_result.ialmax?,
         service_provider_name: decorated_sp_session.sp_name,
         sp_session_requested_attributes: sp_session[:requested_attributes],
```

Comment:
Trying to sift through the layers of abstraction, we could also do something like this, which still indirectly falls back to `service_provider_request.requested_attributes` ([source](https://github.com/18F/identity-idp/blob/9aadb9818fe2b5a63a81253db5a8d6a361d7b4a3/app/decorators/service_provider_session.rb#L55-L57)):

```suggestion
        sp_requested_attributes: decorated_sp_session.requested_attributes,
```

Still trying to understand if we really need `service_provider_request` at all though...